lib/inky.ex

```diff
@@ -33,6 +34,21 @@ defmodule Inky do
   #
   # API
   #
+  def go, do: 45
+
+  def impress do
+    IO.puts("Starting impression")
+    {:ok, pid} = Inky.start_link(:impression, name: Inky.Foo)
+    IO.puts("Started... #{inspect(pid)}")
+
+    IO.puts("Seting pixels...")
+    Inky.set_pixels(Inky.Foo, %{})
+  end
```

Comment:
Looks like a dev function snuck its way in.

Reply:
Yep, removing this block during my cleanup.